rest/route/project_copy.go

```diff
@@ -115,14 +118,30 @@ func (p *copyVariablesHandler) Run(ctx context.Context) gimlet.Responder {
 		varsToCopy.PrivateVars = map[string]bool{}
 	}
 
-	// return the variables that would be copied
+	// Return the variables that would be copied
 	if p.opts.DryRun {
 		return gimlet.NewJSONResponse(varsToCopy)
 	}
 
-	if err := data.UpdateProjectVars(copyToProject.Id, varsToCopy, p.opts.Overwrite); err != nil {
+	if err := data.UpdateProjectVars(copyToProjectId, varsToCopy, p.opts.Overwrite); err != nil {
 		return gimlet.MakeJSONInternalErrorResponder(errors.Wrapf(err, "copying project vars from source project '%s' to target project '%s'", p.copyFrom, p.opts.CopyTo))
 	}
 
 	return gimlet.NewJSONResponse(struct{}{})
 }
+
+func getProjectOrRepoId(identifier string) (string, gimlet.Responder) {
```

Comment:
Should this just return a `gimlet.ErrorResponse` rather than a `gimlet.Responder`?